vhs-frontend/src/components/Header.tsx

```diff
@@ -1,6 +1,7 @@
 import { useState } from 'react';
 import { Link } from 'react-router-dom';
 import { BlastLogo, Hamburger } from '@assets';
+import { t } from 'i18next';
```

Comment:
We don't import the `t` function when we're inside of the component. In components, you should use `useTranslation()` hook.